### ISSUE TITLE
Implement simpler print syntax: `print("Hello World!")`

### ISSUE THREAD

**Issue by certik** [j3-fortran/fortran_proposals#335]
Proposal: the syntax `print("Hello World!")` would be equivalent to `print *, "Hello World!"`.

Advantages:
* Easier to type and remember, as well as to explain to newcomers
* Equivalent to Python's syntax
* Possibly with j3-fortran/generics#136 one could use f-strings for formatting, thus rarely need the more general Fortran formatting capability

Disadvantages (to some):
* It's just syntactic sugar

Discussed at:
* https://fortran-lang.discourse.group/t/why-are-parens-required-around-format-strings/8323/14

**Comment by klausler** [comment]
This would obviously be ambiguous with current conforming and universally portable syntax.

```
character(20)::a='("hello")'
print(a)
end
```

**Comment by certik** [comment]
@klausler you are right, I forgot. I put a link to your example as the main disadvantage.

**Comment by klausler** [comment]
F'2023 notwithstanding, I think that it is a horrible thing to try to change the meanings of existing conforming portable programs.  Please find another syntax or just stick with `print *, ...`.

**Comment by certik** [comment]
@klausler I agree, I wasn't trying to change meaning to existing programs. I forgot that you can put parentheses after `print`, so I thought the parentheses would be enough to disambiguate it. Since this is conflicting with existing syntax, it seems it's not workable.

**Comment by oxcrow** [comment]
I find `print *, "hello world"` significantly easier to use (due to using Fortran for few years)

Although, I agree a simpler syntax maybe helpful to some newcomers.

So, yes, it could be a good addition to Fortran.

**Comment by mjklemm** [comment]
The part about having '{}' as format placeholder could be achieved by this: `print '(Some value {}, another value {})', a, b`. Still not Python syntax, but close.
